src/utils/client.js

```diff
@@ -103,8 +103,8 @@ const createNewsItem = async (news) => {
   client.resetStore();
 };
 
-const updateNewsItem = (id, news) => {
-  client.mutate({
+const updateNewsItem = async (id, news) => {
```

Comment:
First of all, move this code to another file, in `news` folder, call it something like `news.operations.js`. File `client` should only configure client and export it.

Reply:
Also, please, return the result of the client's call from this function to get updated fields back